### DISCUSSION TITLE
Hama TRV schedule?

### DISCUSSION THREAD

**Discussion by Pokerpavian** [Koenkk/zigbee2mqtt#10796] [Devices]
hi
sorry for my bad english. i am new in the smart home "buisness". i have some hue lights in my living room and a few weeks ago i thought about some smart thermostat for the radiators.
so i bought hama trv and a sonoff stick for my raspi pi 3.

the raspi works with home assistant and i installed the the zigbee2mqtt addon. the paring with the trv works.

now its for me possible to set up the temperature manuelly in zigbee2mqtt. but is there a an option to make a schedule for the week?

**Comment by M4RiO-priv** [comment]
I am owner of Hama TRVs as well I'd love to see solution
